include/SampleBuffer.h

```diff
@@ -310,6 +310,14 @@ public slots:
 		ch_cnt_t & channels,
 		sample_rate_t & samplerate
 	);
+#endif
+#ifdef LMMS_HAVE_MP3IMPORT
+	f_cnt_t decodeSampleMP3(
+		QString fileName,
+		sample_t * & buf,
+		ch_cnt_t & channels,
+		sample_rate_t & samplerate
+	);
```

Comment:
```suggestion
```
As I already mentioned, ``decodeSampleSF`` uses sndfile and we don't need this function because of that.